# Detailing Jupiter Release Notes on compatible and non-compliant

werner-roman opened the pull request:
- added section of Non-compliant versions
- further clarification what "compatible" means

cc: @catenax-eV/cx-team-operating-model

Commit:
# Detailing Jupiter RElease Notes on compatible and non-compliant versions
- added section of Non-compliant versions
- further clarification what "compatible" means

jSchuetz88 commented on src/pages/release-notes.md:
```diff
@@ -47,6 +49,12 @@ By default, the versions listed in the [Tractus-X changelog](https://eclipse-tra
 [Latest] – latest released version  
 [Recommended] – recommended by the Catena-X association (Compliant with the [Catena-X Standards](/docs/next/standards/overview) and [End-to-End](docs/next/working-model/release-management/development-and-release/e2e-test) tested)
```
In my understanding, there is no formal check whether a component is actually compliant to a standard or not?

werner-roman replied:
Hey @jSchuetz88,

if there is an idea or proposal on a formal check happy to introduce it immediately. As of now, we would have gone with a "confidence check / official statement" from the respective Expert Group/Comittee.

So they are responsible for it.

Commit:
# chage working from "compliant" to "compatible"

- Chnanged the working at there is no gurantee that a ref. impl ist "compliant" and there is no bindung gurantee from the e.V. and if a company is usinf a ref. impl. they still need to get certified

stephanbcbauer committed:
Merge branch 'main' into chore/detialing_jupiter_release_notes

jSchuetz88 reviewed (approved):
Looks good from my side.
